integration/helpers.go

```diff
@@ -152,3 +153,14 @@ func BuildBaseEvent(namespace string) eventtypes.Event {
 		CommonData: BuildCommonData(namespace),
 	}
 }
+
+func GetTestPodIP(ns string, podname string) string {
+	cmd := exec.Command("kubectl", "-n", ns, "get", "pod", podname, "-o", "jsonpath='{.status.podIP}'")
+	r, err := cmd.Output()
+	if err != nil {
+		return ""
```

Comment:
nit: `return fmt.Sprintf("failed to get Pod IP: %s", err)`. 

Ideally it should return an error but it becomes to annoying to handle it over all the places, so having that will allow us to understand when it fails.